src/series/xyBase.js

```diff
@@ -24,8 +24,8 @@ export default function() {
         return yScale(yValue(d, i));
     };
     base.defined = function(d, i) {
-        return !isNaN(x0Value(d, i)) && !isNaN(y0Value(d, i)) &&
```

Comment:
Yeah - fair point. Changed to `!= null` to allow for undefined and null for 'undefined' values.